# Command arm joint positions

float32 joint_1
float32 joint_2
float32 joint_3
float32 joint_4
float32 joint_5
bool plan
bool wait
bool check_obstruction
---
bool result
